app/controllers/api/v2/auth_source_ldaps_controller.rb

```diff
@@ -54,12 +54,31 @@ def update
         process_response @auth_source_ldap.update_attributes(params[:auth_source_ldap])
       end
 
+      api :GET, "/auth_source_ldaps/:id/test_connection/", N_("Testing LDAP connection")
```

Comment:
a) the method should be PUT, I'd say.  It's more similar to a change/action against the auth source.
b) the action could simply be "test", not sure "connection" is needed
c) "Test", not "Testing"